### DISCUSSION TITLE
Remove redundant files or improve documentation around specific use case

### DISCUSSION THREAD

**Discussion by albertothedev** [vercel/turborepo#3621] [Ideas]
Note: when I say "documentation" I am specifically referring to the "Getting Started" section, with the [Create a New Monorepo](https://turbo.build/repo/docs/getting-started/create-new) subsection being of most relevance, as it is the one that delves deepest into explaining the starter template's setup.

From the answer and other comments to the question #3381 it has become clear that there are two issues with the current implementation:

1. **The example code is not exemplary**. It is supposed to illustrate how to append rules to the custom configuration's in specific projects but it doesn't do this. It instead just extends the configuration, adding no more rules, leading to those ESLint files being entirely redundant and causing confusion as to what their purpose is.
2. **The documentation doesn't explain it**. Even if the example code in those files correctly illustrated the use case, the documentation should still explain how it's doing it and what the reason is.

These issues lead me to three possible solutions:

1. **Delete the files**. This is the easiest to do; simply delete the redundant files and avoid confusion.
2. **Delete the files but also explain the use case in the documentation**. Same as above, but explain in the documentation how to do what they were supposed to show, preferably with an inline code example that includes at least one rule.
3. **Refactor the files so that they actually illustrate the use case and explain it in the documentation**.
    1. **Delete one of the files**. This would make it clear that this setup is about the use of project-specific ESLint configurations that extend the custom one. Having more than one would be redundant and with both files currently being only in `apps` subdirectories, a user might mistake this setup as exclusive to those.
    2. **Add one simple rule (e.g., [quotes](https://eslint.org/docs/latest/rules/quotes)) to the other**. Note that this would slightly change the styling/formatting of that one project.
    3. **Explain the setup in the documentation**.

I might be able to submit a PR once a solution has been agreed on.

@tknickman @mehulkar

**Comment by mehulkar** [comment]
Hey @albertothedev, apologies for the delayed reply and thank you for your patience. Option 1 would be good enough. Feel free to make that PR!
